yarn.lock

```diff
@@ -1,2900 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
we are no more locking packages?

Reply:
Makes no sense for a library, only end product. When somebody installs the library lockfile is not respected. If you remove lockfile you will get more sane (only when needed) dependabot updates